src/Microsoft.ApplicationInsights.AspNetCore/Extensibility/Implementation/Tracing/AspNetEventSource.cs

```diff
@@ -140,6 +140,12 @@ public void UnableToFindModuleToConfigure(string moduleType, string appDomainNam
             this.WriteEvent(11, moduleType, this.ApplicationName);
         }
 
+        [Event(12, Message = "Unable to find QuickPulseTelemetryModule in service collection. QuickPulse/LiveMetrics feature will not be available.", Level = EventLevel.Error, Keywords = Keywords.Diagnostics)]
```

Comment:
pick a name =) Either quick pulse or Live Metrics. Also suggest a fix, Make it easy to support the product